api/models/release.go

```diff
@@ -488,6 +489,20 @@ func (r *Release) resolveLinks(app App, manifest *manifest.Manifest) (*manifest.
 
 			port := other.Ports[0]
 
+			if other.Exports["LINK_PORT"] != "" {
+				i, _ := strconv.Atoi(other.Exports["LINK_PORT"])
+				if err != nil {
+					// handle error
+					fmt.Println(err)
+				}
+
+				for _, p := range other.Ports {
+					if i == p.Container {
+						port = p
+					}
+				}
```

Comment:
Anything we should do if they've set `LINK_PORT` but it doesn't match any of the exposed ports?